demo-circuit/src/constants/mod.rs

```diff
@@ -174,54 +174,13 @@ pub const PHANTOM_FIELD_ELEMENT: FieldElement = field_new!(
     ])
```

Comment:
I don't think we need PHANTOM_FIELD_ELEMENT anymore. Let's replace it with FieldElement::default() in all the places where it's used

Reply:
Done with commit [cd72c64b705c3fc92f5fe40a8e540bbdeb9d523f](https://github.com/HorizenOfficial/zendoo-sc-cryptolib/commit/cd72c64b705c3fc92f5fe40a8e540bbdeb9d523f).